libc/test/src/math/performance_testing/SingleInputSingleOutputPerf.h

```diff
@@ -10,6 +10,7 @@
 #include "src/__support/macros/config.h"
 #include "test/src/math/performance_testing/Timer.h"
 
+#include <algorithm>
```

Comment:
use `src/__support/CPP/algorithm.h` for `min/max` instead.

Reply:
Done.